homeassistant/components/aranet/sensor.py

```diff
@@ -161,7 +166,11 @@ def sensor_update_to_bluetooth_data_update(
         val = getattr(adv.readings, key)
         if val == -1:
             continue
-        val *= desc.scale
+        val = (
```

Comment:
Instead of this, can we set the `translation_key` add [add translations for the states](https://developers.home-assistant.io/docs/internationalization/core/#state-of-entities)?
Also, isn't the user expecting to see "Red/Yellow/Green", based on the official app and device screen?

Reply:
I agree, I think we should represent it same as the app does, also for the translations (was actually waiting to have time to search for example for the translations later, thanks @abmantis)

Reply:
Sure, I'll add translations for the states instead.

> Also, isn't the user expecting to see "Red/Yellow/Green", based on the official app and device screen?

I'm not sure if this would necessarily be the expectation. See https://github.com/home-assistant/core/pull/137291#discussion_r1945887115.

Reply:
I'm not sure if I did the translations correctly, and I couldn't figure out how to test the translations programmatically. Could someone take a look at the latest changes and let me know if you have any further suggestions?